app/helpers/reminder_helper.rb

```diff
@@ -4,22 +4,31 @@ module ReminderHelper
   def print_reminders(data)
     reminders = ''
 
-    if data.active_reminders&.any?
-      reminders = format_reminders(data.active_reminders.map(&:name))
+    if reminders?(data)
+      names_arr = data.active_reminders.map(&:name)
+      reminders = format_reminders(names_arr)
     end
 
     reminders.html_safe
   end
 
   private
 
+  def reminders?(data)
```

Comment:
can this be renamed so it's a bit more clear as to the intent of the method? This reads to me as "does reminders include this data" and it took a little bit to unpack it

Reply:
It was called `has_reminders?` and rubocop yelled at me to remove the has on methods returning bools

Reply:
yeah, rubcop will do that, since this is checking for active reminders maybe `data_has_active_reminders?` or something like that?

Reply:
Or maybe change the parameter to be something a bit more descriptive than `data` (tbh I'm not too familiar with the code so `data` might be the most descriptive name for it), but something to make this a bit more human readable

Reply:
Ya I didnt look too far into why rubocap complained yet. I only hopped on this repo yesterday. Changing the `data` name would be getting a little out of the scope of the issue since this was just for specs. I kinda pushed it refactoring some of the helper to make the specs more concise but I agree `data` is a bit ambiguous. It appears that data can either me medications or strategies. All for debating the semantics of the method name I made though lol

Reply:
I just jumped into the repo yesterday as well! Rubocop usually yells about methods that being with `has`, since this appears to be checking if data has any active reminders, maybe this should be `active_reminders?(data)` or `contains_active_reminders?` or `active_reminders_present?`, something to convey that this is checking that the data argument contains active reminders

Reply:
haha nice. I diden even realize it was hacktoberfest, GH has been a disorganized mad house this week 😂 

I think `active_reminders?` is totally appropriate and will appease rubocop 👍